app/controllers/FileUploadController.scala

```diff
@@ -223,33 +225,35 @@ class FileUploadController @Inject()(fileUploadConnector: FileUploadConnector,
   }
 
   private def extractErrorReason(uploadResponse: Option[UploadResponse]):String ={
-    uploadResponse match {
+		val fileUploadEmptyFileReason= "Envelope does not allow zero length files, and submitted file has length 0"
+
+		uploadResponse match {
       case Some(response) =>
         response.code match {
-          case "400" if response.reason.getOrElse("").contains(Messages("file.upload.empty.file.reason")) =>
+          case "400" if response.reason.getOrElse("").contains(fileUploadEmptyFileReason) =>
```

Comment:
any reason this is no longer in the messages?

Reply:
I've had issues with keys being converted to messages well before they need to e.g. having welsh being fed into a function. I tend to remove Messages to the view and have keys be passed around until that point.

Reply:
yeah, i'd agree but the fileUploadEmptyFileReason is a lengthy string, looks like something from a message file